File: aspnet-version/backend/PetConnectAlmost/DTOs/CommentDto.cs
namespace PetConnectAlmost.DTOs
{
    public class CommentDto
    {
        public long CommentId { get; set; }
        public long UserId { get; set; }
        public string UserFullName { get; set; } = string.Empty;
        [System.Text.Json.Serialization.JsonPropertyName("userProfileImageUrl")]
        public string? UserProfileImageUrl { get; set; }
        public string Content { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}

File: springboot-version/backend/src/main/java/com/petconnect/dto/CommentDto.java
package com.petconnect.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CommentDto {
    private Long commentId;
    private Long postId;
    private Long userId;
    private String userFullName;
    private String userProfileImageUrl;
    private String content;
    private LocalDateTime createdAt;
}
